modules/spring-web-test-client/pom.xml

```diff
@@ -82,6 +82,12 @@
             <artifactId>spring-webflux</artifactId>
             <version>${spring.version}</version>
         </dependency>
+        
+        <dependency>
+            <groupId>org.synchronoss.cloud</groupId>
+            <artifactId>nio-multipart-parser</artifactId>
```

Comment:
Why is this needed?

Reply:
It was required for webflux to support multipart file upload.